addons/aircraft/CfgWeapons.hpp

```diff
@@ -16,8 +16,16 @@ class CfgWeapons {
         magazineReloadTime = 0.1;
     };
 
-    // bigger mag for comanche
     class CannonCore;
+    // Fix attrocious A-10 Cannon Dispersion
+    class Gatling_30mm_Plane_CAS_01_F: CannonCore {
+        class LowROF: Mode_FullAuto {
+            burst = 1;
+            reloadtime = 0.015;
+            dispersion = 0.0028; //0.279508497 = 0.25 * sqrt(0.8^-1); (80%, 5mil. https://en.wikipedia.org/wiki/GAU-8_Avenger#Accuracy) - Luke
```

Comment:
Huh, where does this formula come from?

Reply:
`The GAU-8/A accuracy when installed in the A-10 is rated at "5 mil, 80 percent", meaning that 80 percent of rounds fired will hit within a cone with an angle of five milliradians; this equates to a 40-foot (12-meter) diameter circle at the weapon's design range of 4,000 feet (1,200 m).`

If 80% of all rounds fall within 5 mils, then dispersion is set such that 80% of the area of a given circle is within 5 mils. 
5mils = +/- 0.0025 radian; 
area of circle = pi * r^2; 
area must equal 1/0.8; 
r = 0.0025 *sqrt(0.8^-1);

Reply:
π is gone in your formula.
Formula of type `sqrt (x^-1)` should be transposed to `1 / sqrt x`.

I got
r = 1/√(0.8*π)

Are we sure Arma dispersion goes up and down? (Thus halfing the 5 mils effectively).

Are we sure this Wikipage is talking about 1 circle = 6400 mils and not 2π*2000 = 6283 mils? I suppose that one makes little difference.

Reply:
But π is present in both circles. So let us use a distance measure as a replacement of an angular measure. So, 5 mils is (basically) 5 meters at 1000 meters, so we'll say 5 meters for our radius in our '80%' circle (K) when compared to our dispersion circle (C).

`π(r_C)^2 = 0.8 * π(r_K)^2
(r_C)^2 = 0.8 * (r_K)^2
(r_C)^2 / 0.8 =(r_K)^2
√[(r_C)^2 / 0.8] = r_K
r_C/√[0.8] = r_K
;Substitute 0.005 for r_C;
0.005/√[0.8] = r_K
;Substitute 0.8944 for √[0.8];
0.005/0.8944 = r_K
;Solve;
0.00559 = r_K`

So you, the dispersion *should* be doubled, as the above is radial value. I'll get on that.